### ISSUE TITLE
Add blurbs, remove spelling errors, remove broken links

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3942]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)| @jesstess reported|
|-|-|
|Trac ID|trac#3942|
|Type|enhancement|
|Created|2009-07-30 18:03:45Z|

Added blurbs next to examples that didn't have one. Also fixed "Miscellaneous" spelling, removed broken links to example.html and row_xml.py, and grouped together some of the client/server pairs that weren't next to each other.

Attachments:

* [core-examples-index.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/558/55867721c0977cef542b6276dd07b7492b35b3bc/9efa42de406e488706ab560607d6fce1391f52af.patch) (5183 bytes) - added by jesstess on 2009-08-02 15:54:23Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3942 3942
type__enhancement enhancement
reporter__jesstess jesstess
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1248977025000000 1248977025000000
changetime__1249477624000000 1249477624000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_1"></a>|@jonathanj set owner to @jesstess|
|-|-|

Thanks for this patch, it's always nice to see the documentation being improved.

I have just a few points I'd like to mention:

 1. The expansion of acronyms seems a little haphazard (although it probably has been since forever.) For example, `AMP` seems to be sometimes expanded and `PB` seems to be seldom (never?) expanded. Perhaps expanding the acronyms only in their relevant section headers would read easier than expanding the relevant term for each and every example listing.

 2. `row_xml.py` seems to have gone missing, is this intentional?

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess commented|
|-|-|

Thanks for the feedback.

I replaced the patch with one that doesn't expand AMP and also added the pb_exceptions description I somehow missed.

I removed the link to row_xml.py because it doesn't exist in doc/core/examples and didn't want the broken link. It looks like it just didn't make it over the switch from doc/examples to doc/*/examples, but it uses the deprecated xmlreactor, which looks like it shouldn't have been floating around anywhere since http://twistedmatrix.com/trac/ticket/661 anyway.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jonathanj|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_5"></a>|@jonathanj removed owner|
|-|-|

Okay, I think this looks good now. I'm going to unassign the ticket, to let some other eyes have a look.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh commented|
|-|-|

Looks good to me :-) Fixing the broken link + spelling fix should definitely be in, the blurb is a nice addition.

I'm not sure if more people should review or not, but I think it's okay for inclusion.

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_7"></a>|@jonathanj set status to `closed`|
|-|-|

(In [27221]) Apply core-examples-index.patch.

Author: jesstess
Reviewer: jonathanj, lvh
Fixes: [#3942](https://github.com/twisted/twisted/issues/3942)

Add blurbs to the examples that didn't have one, group client and server
example pairs together and other minor editorial improvements.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
